--- /usr/local/lib/python3.9/site-packages/obol/main.py 2023-08-23 12:01:35.188026039 +0200
+++ /root/main.py       2023-08-23 12:01:05.603621004 +0200
@@ -1,13 +1,13 @@
 #!/usr/bin/python
 import sys
-import logging
+#import logging
 
 from cliff.app import App
 from cliff.commandmanager import CommandManager
 
 import ldap
 import os
-from ConfigParser import ConfigParser
+from configparser import ConfigParser
 
 from . import user, group
 
@@ -76,11 +76,11 @@
                             default=self.default('host', 'ldap://localhost'))
         parser.add_argument('-b', metavar='BASE_DN',
                             default=self.default('baseDN', 'dc=local'))
-        loglevels = [key for key in logging._levelNames
-                     if isinstance(key, str)]
+#        loglevels = [key for key in logging._levelNames
+#                     if isinstance(key, str)]
 
-        parser.add_argument('--logLevel', choices=loglevels,
-                            default=self.default('logLevel', 'INFO'))
+#        parser.add_argument('--logLevel', choices=loglevels,
+#                            default=self.default('logLevel', 'INFO'))
         return parser
 
     def prepare_to_run_command(self, cmd):
